mep/books/views.py

```diff
@@ -1,25 +1,85 @@
 from dal import autocomplete
 from django.db.models import Q
-from django.views.generic import DetailView, ListView
 from django.urls import reverse
+from django.views.generic import DetailView, ListView
+from django.views.generic.edit import FormMixin
 
+from mep.books.forms import ItemSearchForm
 from mep.books.models import Item
 from mep.books.queryset import ItemSolrQuerySet
-from mep.common.views import LabeledPagesMixin, RdfViewMixin
-from mep.common.utils import absolutize_url
 from mep.common import SCHEMA_ORG
+from mep.common.utils import absolutize_url
+from mep.common.views import (AjaxTemplateMixin, FacetJSONMixin,
+                              LabeledPagesMixin, RdfViewMixin)
 
 
-class ItemList(LabeledPagesMixin, ListView, RdfViewMixin):
+class ItemList(LabeledPagesMixin, ListView, FormMixin, AjaxTemplateMixin, FacetJSONMixin, RdfViewMixin):
```

Comment:
That's a lot of mixins! And it looks like there's a lot of overlap in view functionality with our member list. But I'm ok with leaving it here for now until we have a better idea of what is actually common.